sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/Error.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.keyvault.administration.implementation.models;
+
+import com.azure.core.annotation.Immutable;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The Error model. */
+@Immutable
+public final class Error {
```

Comment:
In the Java world, it is common practice to assume things inside an implementation folder to be internal, since there is no way to access internal classes in different packages without making them public. Additionally, in this case everything in the `administration/implementation/models` folder was generated with AutoRest. Custom models are located inside `administration/models`.

Reply:
This would be exposed through `KeyVaultErrorException`.